subsystem/metrics_test.go

```diff
@@ -895,6 +897,36 @@ var _ = Describe("Metrics tests", func() {
 			// check generated events
 			assertHostValidationEvent(ctx, clusterID, string(*h.ID), models.HostValidationIDNtpSynced, false)
 		})
+		It("'sufficient-network-latency-requirement-for-role' generated", func() {
```

Comment:
Maybe you can put those tests in a single context and put the common part in before each?

Reply:
Sounds good.

Reply:
Done.